README.md

```diff
@@ -300,3 +301,45 @@ For example:
 
 More info in https://golang.org/pkg/testing
 
+## Spread tests
+
+We have a set of spread (https://github.com/snapcore/spread) tests which
+can be executed on a virtual machine or real hardware.
+
+In order to run those tests you need the follow things
+
+* ubuntu-image
+* spread
+
+ You can install ubuntu-image snap
+
+    snap install --edge --devmode ubuntu-image
+
+NOTE: Because of confinement, snap spread cannot access virtualized environment. 
+For now, then, let's install spread from sources, this way:
+
+    WORKDIR=`mktemp -d`
```

Comment:
not sure why we install to tmpdir. is installing to normal GOPATH preferable to simplify running spread tests again later?